docsteady/__init__.py

```diff
@@ -203,6 +203,21 @@ def generate_report(format, username, password, trace, plan, path):
 
     file = open(path, "w") if path else sys.stdout
     print(_as_output_format(text), file=file or sys.stdout)
+# output the plan - TR without results
+    template = env.get_template(f"tpnoresult.{Config.TEMPLATE_LANGUAGE}.jinja2")
+    metadata["template"] = template.filename
+    text = template.render(metadata=metadata,
+                           testplan=plan_dict['tplan'],
+                           testcycles=list(testcycles_map.values()),  # For convenience (sorted)
+                           testcycles_map=testcycles_map,
+                           testresults=[],  # no results lpan
+                           testresults_map={},  # no results for plan
+                           attachments=plan_dict['attachments'],
+                           testcases_map=testcases_map)
+
+    path = path.replace(".tex", "plan.tex")
+    file = open(path, "w") if path else sys.stdout
```

Comment:
This file is never closed. Using `with` here is a bit of a pain but at the very least there should be something like:

```python
if file != sys.stdout:
    file.close()
```
after the `print()`.